client-app/app/templates/components/back-to-site-link.hbs

```diff
@@ -0,0 +1,7 @@
+{{#if this.shouldDisplay}}
+  <div id="back-to-site-panel">
+    <a href={{@path}} rel="noopener noreferrer">
+      {{@text}}
```

Comment:
It might be nice to add an `arrow-left` icon here, similar to what we do in core.